roles/openshift_master/templates/master.yaml.v1.j2

```diff
@@ -139,6 +139,7 @@ networkConfig:
 {% endif %}
 # serviceNetworkCIDR must match kubernetesMasterConfig.servicesSubnet
   serviceNetworkCIDR: {{ openshift.common.portal_net }}
+  externalIPNetworkCIDRs: {{ openshift.master.sdn_cluster_network_cidrs }}
```

Comment:
Actually, this should be the openshift_master_facts line and the jinja template should be here in the config.